internal/dev_server/db/sqlite.go

```diff
@@ -49,6 +49,39 @@ VALUES (?, ?, ?, ?, ?)
 	return err
 }
 
+func (s Sqlite) UpsertOverride(ctx context.Context, override model.Override) error {
+	valueJson, err := override.Value.MarshalJSON()
+	if err != nil {
+		return errors.Wrap(err, "unable to marshal override value when writing override")
+	}
+
+	_, err = s.database.Exec(`
+		INSERT INTO overrides (project_key, flag_key, value, active)
+		VALUES (?, ?, ?, ?)
+			ON CONFLICT(flag_key, project_key) DO UPDATE SET
+			    value=excluded.value,
+			    active=excluded.active,
+			    version=version+1;
+	`,
+		override.ProjectKey,
+		override.FlagKey,
+		valueJson,
+		override.Active,
+	)
+
+	return err
+}
+
+func (s Sqlite) DeleteOverride(ctx context.Context, projectKey, flagKey string) error {
+	_, err := s.database.Exec(`
+		UPDATE overrides set active = false where project_key = ? and flag_key = ?
```

Comment:
This should also increment the version. This is needed so that the sdk will pick up the change (it ignores changes if the version number isn't bigger).

Reply:
oh duh my b!